From 71eedfeb86445606febbc8f255a0fe2dec4b47f2 Mon Sep 17 00:00:00 2001
From: Wulf C. Krueger <philantrop@exherbo.org>
Date: Sun, 17 Jan 2010 15:48:08 +0100
Subject: [PATCH] Disable spamming syslog.
Upstream: No, they're going to change this whole thing with 2.4 in April.

---
 imap/global.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/imap/global.c b/imap/global.c
index 5d64895..aa11a72 100644
--- a/imap/global.c
+++ b/imap/global.c
@@ -566,7 +566,7 @@ int access_ok(const char *user, const char *service, char *msgbuf, int size)
 	const char *data = NULL;
 	int datalen;
 
-	syslog(LOG_DEBUG, "fetching user_deny.db entry for '%s'", user);
+	//	syslog(LOG_DEBUG, "fetching user_deny.db entry for '%s'", user);
 	do {
 	    r = DENYDB->fetch(db, user, strlen(user), &data, &datalen, NULL);
 	} while (r == CYRUSDB_AGAIN);
-- 
1.6.6

